variables.tf

```diff
@@ -4,29 +4,52 @@ variable "cluster_name" {
 }
 
 variable "base_domain" {
-  description = "The base domain used for Ingresses."
+  description = "The base domain used for Ingresses. If not provided, nip.io will be used taking the NLB IP address."
```

Comment:
I agree that maybe creating the domain itself inside the module could be a bad idea, I did not consider that.

As for the use of nip.io, we already did the same on the EKS cluster, and the DNS record is not created unless you pass the `base_domain` variable (the domain is still created outside, as far as I know).

Reply:
Ok, then this would need to be discussed more globally for all cluster modules. So for now it's fine to make this config optional to be consistent across clusters.